tests/integration/transactions/test_trust_set.py

```diff
@@ -10,7 +13,7 @@ class TestTrustSet(IntegrationTestCase):
     @test_async_and_sync(globals())
     async def test_basic_functionality(self, client):
         issuer_wallet = Wallet.create()
-        response = await submit_transaction_async(
+        response = await sign_and_reliable_submission_async(
```

Comment:
If this function is necessary now for tests involving submitting transactions, can you add it to the CONTRIBUTING as a part of writing tests?